File: judge-runtime/spec/lang.elixir/validator.ex
defmodule Validator do
  @spec swap_first_and_last_words(String.t) :: String.t
  def swap_first_and_last_words(s) do
    words = String.split(s)

    if length(words) <= 1 do
      s
    else
      [first | rest] = words
      last = List.last(rest)
      rest = Enum.take(rest, length(rest) - 1)
      [last | rest] ++ [first]
      |> Enum.join(" ")
    end
  end
end

File: judge-runtime/spec/lang.cs/validator.cs
using System;

public class Validator {
    public string SwapFirstAndLastWords(string s) {
        var words = s.Split(' ');

        if (words.Length <= 1) {
            return s;
        }

        (words[0], words[^1]) = (words[^1], words[0]);

        return string.Join(" ", words);
    }
}
